### DISCUSSION TITLE
Does this javacv library open camera of end user who is accessing web application via browser

### DISCUSSION THREAD

**Discussion by ssingh-cls** [bytedeco/javacv#2100] [Q&A]
Hi, 
 I need to integrate this javacv in web application being accessed by users via browser and not in stand alone type application.

In that case, could someone please confirm that this javacv library will open end user's desktop/laptop camera (and not the server's where application is deployed) when user is accessing this web application via browser? Thanks!

**Comment by saudet** [comment]
Today's browsers don't typically support Java...
